cvat/apps/dataset_manager/formats/market1501.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import zipfile
+from tempfile import TemporaryDirectory
+
+from datumaro.components.dataset import Dataset
+from datumaro.components.extractor import AnnotationType, Label, LabelCategories
+
+from cvat.apps.dataset_manager.bindings import CvatTaskDataExtractor, \
+    import_dm_annotations
+from cvat.apps.dataset_manager.util import make_zip_archive
+
+from .registry import dm_env, exporter, importer
+
+
+@exporter(name='Market-1501', ext='ZIP', version='1.0')
+def _export(dst_file, task_data, save_images=False):
+    dataset = Dataset.from_extractors(CvatTaskDataExtractor(
+        task_data, include_images=save_images), env=dm_env)
+    with TemporaryDirectory() as temp_dir:
+        for item in dataset._data._source:
```

Comment:
This should not be done this way, use transform for this.

Reply:
I couldn't find transform for this task. What should I use here?